### DISCUSSION TITLE
It either doesn't exist yet, or may not use project.json for configuration. Use `addProjectConfiguration()` instead if you want to create a new project.

### DISCUSSION THREAD

**Discussion by sonikasharma1403** [nxext/nx-extensions#1025] [Q&A]
Error:
It either doesn't exist yet or may not use project.json for configuration. Use `addProjectConfiguration()` instead if you want to create a new project.

INFO:
get this error while using the Nx generator in an angular CLI application. This error is thrown when we try to either do `addProjectConfiguration` or `updateProjectConfiguration`. Note that the generator has been converted using the `convertNxGenerator`.

The NX app works perfectly fine but facing issues while making it compatible with the angular App. 

Could someone please help me with this..

**Comment by DominikPieper** [comment]
Which plugin are you using? We usually don’t test against Angular cli, just with the Nx cli
